content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -436,12 +436,12 @@ A ServiceAccount for `kube-proxy` is created in the `kube-system` namespace; the
 
 #### DNS
 
-- In Kubernetes version 1.18 kube-dns usage with kubeadm is deprecated and will be removed in a future release
+- In Kubernetes version 1.21 kube-dns usage with kubeadm has been removed
```

Comment:
I'd write:
```suggestion
- In Kubernetes version 1.21 support for using `kube-dns` with kubeadm was removed.
  You can use CoreDNS with kubeadm even where the related Service is named `kube-dns`.
```
and I'd put that statement at the _end_ of the list. Readers are likely to be more interested in what _is_ supported.